### PR TITLE
Added missed locales support and romanian translation

### PR DESCRIPTION
**Pull request by wanderrer**
**Describe Pull request**
Added missed locales support and romanian translation

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] yes
- Does your code fit the style guidelines? [yes/no] yes
- Does your PR fit the contribution guidelines? [yes/no] yes

### CODE REVIEW

**Comment by tom-osborne** [comment]
Just linting issues to fix please @wanderrer

File: locales/en.lua [hunk 1]
```diff
@@ -5,6 +5,19 @@ local Translations = {
         not_enough_money = 'You dont have enough money',
         finish_payments = 'You must finish paying off this vehicle, Before you can sell it..',
         no_space_on_lot = 'There is not space for your car on the lot!'
```
Comment (tom-osborne) [inline locales/en.lua:7]: missing comma at end of line

File: locales/ro.lua [hunk 2]
```diff
@@ -0,0 +1,55 @@
+local Translations = {
+    error = {
+        not_your_vehicle = 'Acesta nu este vehiculul tau..',
+        vehicle_does_not_exist = 'Vehiculul nu exista',
+        not_enough_money = 'Nu ai suficienti bani',
+        finish_payments = 'Trebuie sa achiti de platit ratele, inainte de a putea vinde vehiculul...',
+        no_space_on_lot = 'Din pacate, nu mai sunt spatii pentru a pune masina la vanzare!'
```
Comment (tom-osborne) [inline locales/ro.lua:7]: Missing comma at end of line

File: locales/ro.lua [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    },
+    menu = {
+        view_contract = 'Vezi contractul',
+        view_contract_int = '[E] Vezi contractul',
+        sell_vehicle = 'Vinde vehiculul',
+        sell_vehicle_help = 'Vinde vehiculul tau catre cetateni interesati',
+        sell_back = 'Cumpara inapoi vehiculul!',
+        sell_back_help = 'Cumpara-ti masina inapoi la un pret promotional!',
+        interaction = '[E] Vinde vehicul',
+
```
Comment (tom-osborne) [inline locales/ro.lua:19]: Remove whitespace

**Review by tom-osborne — CHANGES_REQUESTED** [review]

File: locales/en.lua (hunk 1 above)
Comment (wanderrer) [reply locales/en.lua]: was added

File: locales/ro.lua (hunk 2 above)
Comment (wanderrer) [reply locales/ro.lua]: was added

File: locales/ro.lua (hunk 3 above)
Comment (wanderrer) [reply locales/ro.lua]: was removed

**Review by tom-osborne — APPROVED** [review]

**Review by tom-osborne — APPROVED** [review]

**Pull request merged**